keras/src/layers/core/dense.py

```diff
@@ -352,6 +408,65 @@ def _int8_build(self, kernel_shape):
             trainable=False,
         )
 
+    def _gptq_build(self, kernel_shape, config):
+        self.gptq = False
```

Comment:
It's very unintuitive that the result of `_gptq_build()` would be to "turn off" gptq.

Either we need a better name than `self.gptq` here, or at least a comment explaining.